### ISSUE TITLE
Price spoken 

### ISSUE THREAD

**Issue by jojolo** [drpout/boilr#130]
Have the price spoken every X amount of time. Have user pick how many seconds or minutes.

**Comment by algazarra** [comment]
@jojolo are you day trading? never felt the need for such a feature.

**Comment by jojolo** [comment]
Yeah, for day trading. Say a lot of price movement happens middle of the night. Instead of fiddling with the phone and bright screen you can have the price spoken as you lay in bed and try to rest. 

There are other apps that do it and I can always use more than one app. I'm just thinking in terms of 1 app that rules them all.

**Comment by algazarra** [comment]
I see! hmmm I'd like to see how other apps do it. Can you post a link to a good example? thanks for the input!

**Comment by jojolo** [comment]
The app I use is Bitcoin Checker for the price speaking feature. 

https://play.google.com/store/apps/details?id=com.mobnetic.coinguardian&hl=en

https://github.com/mobnetic/BitcoinChecker

**Comment by dllud** [comment]
> Instead of fiddling with the phone and bright screen you can have the price spoken as you lay in bed and try to rest. 

Nice, finally we have a use case.

On the early beginnings of Boilr I was messing around with TTS (Text-To-Speech) on [OsmAnd~](https://github.com/osmandapp/Osmand) and thought it would be a *cool* feature we could copy-paste from them (after all we can't, their [voices are proprietary](https://github.com/osmandapp/Osmand/blob/master/LICENSE)). Then we discovered Bitcoin Checker and I noticed it has TTS. But there was still missing valid a use cases for this and @algazarra always thought of it as frivolous.

**Comment by dllud** [comment]
We could couple this with #102, having the price spoken when the status bar ticker is displayed.

**Comment by algazarra** [comment]
Yes but leave options to deactivate as it can prove too intrusive.
